app/models/patient.rb

```diff
@@ -55,6 +56,7 @@ class Patient
   field :other_phone, type: String
   field :other_contact_relationship, type: String
   field :identifier, type: String
```

Comment:
We're basically subbing out what's populating identifier, right? Should we just use that field rather than adding another one?

Reply:
(If I remember right the reason we weren't doing this was that we want to use both and gradually transition.)